### ISSUE TITLE
Ruby: RuntimeError when following the docs

### ISSUE THREAD

**Issue by DannyBen** [tangramxyz/tangram#20]
When following the "predict with Ruby" instructions [here](https://www.tangram.xyz/docs/getting_started/predict/ruby), it raises a `RuntimeError`

```shell
$ ruby test.rb
/home/vagrant/.rbenv/versions/3.0.1/lib/ruby/gems/3.0.0/gems/tangram-0.6.1/lib/tangram/tangram.rb:424:in `block in new_predict_input': tangram error (RuntimeError)
        from /home/vagrant/.rbenv/versions/3.0.1/lib/ruby/gems/3.0.0/gems/tangram-0.6.1/lib/tangram/tangram.rb:416:in `each'
        from /home/vagrant/.rbenv/versions/3.0.1/lib/ruby/gems/3.0.0/gems/tangram-0.6.1/lib/tangram/tangram.rb:416:in `new_predict_input'
        from /home/vagrant/.rbenv/versions/3.0.1/lib/ruby/gems/3.0.0/gems/tangram-0.6.1/lib/tangram/tangram.rb:406:in `block in new_predict_input_vec'
        from /home/vagrant/.rbenv/versions/3.0.1/lib/ruby/gems/3.0.0/gems/tangram-0.6.1/lib/tangram/tangram.rb:404:in `each'
        from /home/vagrant/.rbenv/versions/3.0.1/lib/ruby/gems/3.0.0/gems/tangram-0.6.1/lib/tangram/tangram.rb:404:in `new_predict_input_vec'
        from /home/vagrant/.rbenv/versions/3.0.1/lib/ruby/gems/3.0.0/gems/tangram-0.6.1/lib/tangram/tangram.rb:277:in `predict'
        from test.rb:29:in `<main>'
```

A couple of side notes that I encountered while getting started with tangram:

1. The DockerHub link on the [installation page](https://www.tangram.xyz/docs/install) is wrong - missing a `/r/` ([fixed link](https://hub.docker.com/r/tangramxyz/tangram))
2. The `tangram app` command does not output anything, took me a few attempts to get it to listen on the correct port, and I didn't know if its running at all. Would be nice if it just stated "Server running on 0.0.0.0:8080".

---

- OS: Ubuntu 18 (Vagrant on Windows)
- Ruby: 3.0.1
- Installed Tangram via the Ubuntu 18 instructions

**Comment by nitsky** [comment]
Hi @DannyBen thank you very much for the bug reports.

> When following the "predict with Ruby" instructions here, it raises a RuntimeError

The exception is raised here: https://github.com/tangramxyz/tangram/blob/db47f34c5c98614227fdbb658e0a793d7db2834e/languages/ruby/lib/tangram/tangram.rb#L424.

The input value for any column must be either a `Float` or a `String`. The example in the docs was missing `.0` on a few of the numeric values. Thank you for catching this, I fixed it. In addition, the error message you are seeing is not descriptive at all. Sorry! I have pushed a commit that improves it. It will be part of the next release.

> The DockerHub link on the installation page is wrong

Fixed. Thank you.

> Would be nice if it just stated "Server running on 0.0.0.0:8080".

Done. This will be part of the next release.
